source/hash_table.c

```diff
@@ -991,6 +991,14 @@ uint64_t aws_hash_ptr(const void *item) {
     return ((uint64_t)b << 32) | c;
 }
 
+uint64_t aws_hash_combine(uint64_t item1, uint64_t item2) {
+    uint32_t b = (uint32_t)item2;
```

Comment:
This conversion could overflow, but it's intended, right? Also, would it make sense to replace it with the following line? 
```c
uint32_t b = item2 & 0xFFFFFFFF; /* LSB */
uint32_t c = item2 >> 32; /* MSB */
```

Reply:
ok